.editorconfig

```diff
@@ -19,7 +19,7 @@ skip = .eggs
 # RST files (used by sphinx)
 [*.rst]
 indent_style = space
-indent_size = 3
+indent_size = 4
```

Comment:
This is the recommended standard for rst (or least what I've been using for the last 20 years). Probably should stay as is.

Reply:
ok